Resources.md

```diff
@@ -0,0 +1,118 @@
+# More resources : Schnorr, Taproot, MAST
+
+A quick [review](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-May/015951.html) of different consensus changes on the mailing list, with
+advantages, drawbacks, among others Schnorr, Taproot, MAST, Grafroot, new sighashes.
+
+# Schnorr : a dlog-based Efficient Signature Scheme
+
+## Introduction
+
+To get started, a high-level list of [Schnorr advantages](https://bitcoin.stackexchange.com/questions/34288/what-are-the-implications-of-schnorr-signatures/35351#35351) like :
+- reduce onchain footprint
+- m-of-n key aggregation
+- n-of-m key aggregation
```

Comment:
Are one of these intended to be "n-of-n" ?

Also, Optech style guidelines suggest using k-of-n instead of m-of-n, in case you'd like to follow them.

Reply:
+1 for "n-of-n"

Mentioned style guide reference: https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#spelling